operators/Makefile

```diff
@@ -341,6 +350,10 @@ ci: dep-vendor-only check-fmt generate check-local-changes unit integration e2e-
 # Let's use n1-standard-8 machine to have enough room for multiple pods on a single node.
 ci-e2e: ci-bootstrap-gke e2e
 
+# Run e2e tests in gke using custom operator image
+ci-e2e-rc: export SKIP_DOCKER_COMMAND=true
+ci-e2e-rc: ci-bootstrap-gke e2e
```

Comment:
I'm not fan to increase the number of `IF` in this Makefile that already seems a bit too complex.
I tried to see if we can avoid the introduction of `SKIP_DOCKER_COMMAND`.
I think another approach could be to not reuse the `deploy` target (that you modified with a `IF` to skip the `docker-build docker-push`) but instead reuse the underlying tasks.
For the `docker-build docker-push` called in the `bootstrap-gke`, we took a shortcut to ease our dev workflow when using GKE (https://github.com/elastic/cloud-on-k8s/pull/437) and it's bad. It seems to me that skipping it is only an optimization? Note: when the process-manager will be removed, we will not need anymore to build and push the latest operator Docker image \o/.

What do you think if we do:
```make
ci-e2e-rc: MANAGED_NAMESPACE=e2e
ci-e2e-rc: ci-bootstrap-gke e2e-setup check-gke install-crds apply-operators e2e-docker-image e2e-run
```

Reply:
It looks like my initial implementation. But @sebgl propose to re-use existed `e2e` target. Which leads to existed stuff 😄

Reply:
Look at https://github.com/elastic/cloud-on-k8s/pull/1135#issuecomment-505790463

Reply:
I think @sebgl wanted to reuse the same e2e job not only the make target to avoid writing another `ci/jobs/xxx.yml` and avoid adding another target in the `build/ci/Makefile` which is almost similar to the existing ones.

The initial implementation you're thinking about is 84a6c26d?

Reply:
> The initial implementation you're thinking about is 84a6c26?

Yes.